custom_components/lennoxs30/__init__.py

```diff
@@ -89,9 +98,6 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
     allergenDefenderSwitch = config.get(DOMAIN).get(CONF_ALLERGEN_DEFENDER_SWITCH)
     app_id = config.get(DOMAIN).get(CONF_APP_ID)
     conf_init_wait_time = config.get(DOMAIN).get(CONF_INIT_WAIT_TIME)
-    ip_address = config.get(DOMAIN).get(CONF_IP_ADDRESS)
-    if ip_address == "None":
```

Comment:
With this removed it’s going to talk to the cloud rather than local.  Looks like a merge issue?  I’m not skilled with git, so if this was my mistake apologies.

Reply:
This logic is being moved up next to the user/password part.

Reply:
Indeed it is 👍